geomstats/geometry/discrete_curves.py

```diff
@@ -1772,66 +1743,73 @@ def space_derivative(curve):
         return space_deriv
 
 
-class QuotientSRVMetric(SRVMetric):
-    """Metric on shape space induced by the SRV metric.
+class SRVShapeBundle(DiscreteCurves, FiberBundle):
```

Comment:
NIT (not necessarily for this PR): I wonder if we should put the name of the group inducing the quotient in the name of the class.

We could have a SRVShapeBundle only by quotienting the action of rotations.

Actually, in these shape manifolds, we will have a series of quotient, for each additional group applied.

Thus: `SRVRotationBundle`, `SRVParamBundle`?

--> I'll ask the question in our issue on DiscreteCurves.